pkg/management/component.go

```diff
@@ -432,8 +432,8 @@ func (c *component) genericCreateUser(ctx context.Context, accessToken string, c
 	}
 
 	var username = ""
-	if user.Username != nil {
-		username = *user.Username
+	if userRep.Username != nil {
```

Comment:
Was this causing a bug ? 

I see there is other places in this function in which we are using user instead of userRep.

Reply:
Not a bug but we still had events without username when there is no reason not to have it